sdk/network/azure-mgmt-network/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 0.0.0 (it should be stable) (2023-03-28)
```

Comment:
please copy the changelog @lmazuel and I wrote over [here](https://github.com/Azure/azure-sdk-for-python/pull/29444/files#diff-710c629ec2e871abb003dc86332beba7dea49965e378d498d6aa22deba454499R3)

Reply:
updated